draft-ietf-ppm-dap.md

```diff
@@ -2391,7 +2391,10 @@ If all of the above checks succeed, the input share is valid.
 The Helper handles the aggregation job initialization synchronously:
 
 ~~~ http
-PUT /helper/tasks/8BY0RzZMzxvA46_8ymhzycOB9krN-QIGYvg_RsByGec/aggregation_jobs/lc7aUeGpdSNosNlh-UZhKA
+========== NOTE: '\' line wrapping per RFC 8792 ===========
```

Comment:
I'm not a fan of the inclusion of this in every example, but it's what the RFC recommends.

The key text is the "NOTE: '\' line wrapping per RFC 8792" part.  How you decorate that to make it clear it's not part of the actual example is up to you.  So this is fine.

Personally, I would probably add something to your "[Conventions and Definitions](https://datatracker.ietf.org/doc/html/draft-ietf-ppm-dap#name-conventions-and-definitions)" section instead.  Just say that all examples use RFC 8792 "\\" wrapping (one backslash).  That works as long as you weren't planning on having a trailing backslash in any figure (I couldn't find one).